docs/ref/models/database-functions.txt

```diff
@@ -623,7 +623,7 @@ that deal with date-parts can be used with ``DateField``::
 ``DateTimeField`` truncation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. class:: TruncDate(expression, **extra)
+.. class:: TruncDate(expression, tzinfo=None, **extra)
```

Comment:
Hey - thanks for this. A couple of comments on the docs from me. 

- Please add a `.. versionchanged:: 3.2` annotation to `TruncDate` and `TruncTime` with a short comment saying `tzinfo` was added. 

- Add an entry in the release notes in `3.2.txt`

https://docs.djangoproject.com/en/3.1/internals/contributing/writing-code/submitting-patches/#new-features

Reply:
@smithdc1 Thanks for pointing that out.  Pushed up those changes.